gamestonk_terminal/etf/screener_model.py

```diff
@@ -0,0 +1,191 @@
+"""Screener model"""
+__docformat__ = "numpy"
+
+import argparse
+from typing import List
+import os
+import configparser
+import pandas as pd
+from tabulate import tabulate
+from gamestonk_terminal.helper_funcs import parse_known_args_and_warn, export_data
+
+
+def etf_screener(other_args: List[str]):
+    """
+    Screens the etfs pulled from my repo, which is updated daily at midnight EST
+    Parameters
+    ----------
+    other_args:
+        List or argparse arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="etfscr",
+        add_help=False,
+        description="Screens ETFS from a personal scraping github repository.  Data scraped from stockanalysis.com",
+    )
+
+    parser.add_argument(
+        "-p", "--min_price", help="min price", dest="min_price", default=False
+    )
+    parser.add_argument(
+        "-P", "--max_price", help="max price", dest="max_price", default=False
+    )
+    parser.add_argument(
+        "-a", "--min_assets", help="min assets ($M)", dest="min_assets", default=False
+    )
+    parser.add_argument(
+        "-A", "--max_assets", help="max assets ($M)", dest="max_assets", default=False
+    )
+    parser.add_argument(
+        "-n",
+        "--min_nav",
+        help="min nav (net asset value)",
+        dest="min_nav",
+        default=False,
+    )
+
+    parser.add_argument(
+        "-N",
+        "--max_nav",
+        help="max nav (net asset value)",
+        dest="max_nav",
+        default=False,
+    )
+    parser.add_argument(
+        "-e", "--min_exp", help="min expense ratio (%%)", dest="min_exp", default=False
+    )
+
+    parser.add_argument(
+        "-E", "--max_exp", help="max expense ratio (%%)", dest="max_exp", default=False
+    )
+
+    parser.add_argument(
+        "-r", "--min_pe", help="min pe ratio", dest="min_pe", default=False
+    )
+    parser.add_argument(
+        "-R", "--max_pe", help="max pe ratio", dest="max_pe", default=False
+    )
+
+    parser.add_argument(
+        "-d", "--min_div", help="min dividend yield (%%)", dest="min_div", default=False
+    )
+    parser.add_argument(
+        "-D", "--max_div", help="max dividend yield (%%)", dest="max_div", default=False
+    )
+    parser.add_argument(
+        "-b", "--min_beta", help="min 5Y beta", dest="min_beta", default=False
+    )
+    parser.add_argument(
+        "-B", "--max_beta", help="max beta", dest="max_beta", default=False
+    )
+    parser.add_argument(
+        "--config",
+        help="Load options from config file",
+        dest="config",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--export",
+        choices=["csv", "json", "xlsx"],
+        default="",
+        dest="export",
+        help="Export dataframe data to csv,json,xlsx file",
+    )
+    # pylint: disable=no-member
```

Comment:
Can we add a "-n/--num" argument to display a maximum of NUM stocks? And add a 20-30 by default, and randomize these if so? 

Otherwise the table is so big that you end up not being able to see the headers. Let me know your thoughts.

Reply:
sure.  I already have an -n so ill just use --num

Reply:
That's perfect!